src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const rentCar = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    return rentCar - LONG_TERM_DISCOUNT;
```

Comment:
This `if` block correctly returns the value for long-term rentals. Following the checklist, there is no need for an `else` statement after this `if` block since `return` will exit the function if the condition is true.